any_run/plugin.spec.yaml

```diff
@@ -0,0 +1,369 @@
+      type: string
+      required: true
+      example: 'https://app.any.run/tasks/0cb5990e-7858-4455-a5f8-8d704051901b'
+    pcap:
+      title: PCAP
+      description: Task PCAP URL
+      type: string
+      required: true
+      example: 'https://content.any.run/tasks/0cb5990e-7858-4455-a5f8-8d704051901b/download/pcap'
+    file:
+      title: File
+      description: Task file URL
+      type: string
+      required: true
+      example: 'https://content.any.run/tasks/0cb5990e-7858-4455-a5f8-8d704051901b/download/files/005d7277-b4c1-4b96-bcd9-ac02b799eec9'
+    json:
+      title: JSON
+      description: Task JSON URL
+      type: string
+      required: true
+      example: 'https://api.any.run/report/0cb5990e-7858-4455-a5f8-8d704051901b/summary/json'
+    misp:
+      title: MISP
+      description: Task MISP URL
+      type: string
+      required: true
+      example: 'https://api.any.run/report/0cb5990e-7858-4455-a5f8-8d704051901b/summary/misp'
+    tags:
+      title: Tags
+      description: Task tags
+      type: '[]string'
+      required: true
+      example: '["suspicious"]'
+    date:
+      title: Date
+      description: Task date
+      type: string
+      required: true
+      example: 2020-04-17T18:44:37.483Z
+    hashes:
+      title: Hashe
+      description: Task hashes
+      type: hashes
+      required: true
+connection:
+  api_key:
+    title: API Key
+    description: API Key for Any.Run
+    type: credential_secret_key
+    required: false
+    example: WbTbwa4KFk77eQNffMJynWXm49jLwGjwPMKM9Xc4
+  credentials:
+    title: Credentials
+    description: Username and password
+    type: credential_username_password
+    required: false
+    example: '{ "username": "user@example.com", "password":"mypassword"}'
+actions:
+  get_history:
+    title: Get History Tasks
+    description: Get history Tasks
+    input:
+      team:
+        title: Team
+        description: Leave this field blank to get your history or specify to get team history
+        type: boolean
+        required: false
+        default: false
+        example: false
+      skip:
+        title: Skip
+        description: Skip records
+        type: integer
+        required: false
+        default: 0
+        example: 25
+      limit:
+        title: Limit
+        description: Limits number of records, Size range 1-100
+        type: integer
+        required: false
+        default: 25
+        example: 25
+    output:
+      tasks:
+        title: Tasks
+        description: History tasks
+        required: false
+        type: '[]task'
+  get_report:
+    title: Get Report
+    description: Get task report
+    input:
+      task:
+        title: UUID
+        description: Task UUID
+        type: string
+        required: true
+        example: '0cf223f2-530e-4a50-b68f-563045268648'
+    output:
+      reports:
+        title: Reports
+        description: Task reports
+        required: false
+        type: 'object'
```

Comment:
Can we create custom types instead of using the object? Users will not be able to use the variable selector in the product with it.

You can auto-generate it with `tools/json_to_spec.py` in the repo. JSON output of successful action runs with icon-plugin are stored in .output directory.

```
$ ../tools/json_to_spec.py .output/action_get_report.json

...
    image:
      title: "Image"
      type: string
      description: "Image"
      required: false
    times:
      title: "Times"
      type: times
      description: "Times"
      required: false


output:
  reports:
      title: "Reports"
      type: reports
      description: "Reports"
      required: false

```